src/functions/template-tags/post.php

```diff
@@ -63,3 +63,32 @@ function tribe_the_content( $more_link_text = null, $strip_teaser = false, $post
 	echo tribe_get_the_content( $more_link_text, $strip_teaser, $post_id );
 }
 
+/**
+ * Wrapper for post_class function that allows us to in-memory cache
+ *
+ * @since TBD
+ *
+ * @param string|string[] $class   Space-separated string or array of class names to add to the class list.
+ * @param int|WP_Post     $post_id Optional. Post ID or post object.
+ *
+ * @return string[] Array of class names.
+ */
+function tribe_get_post_class( $class, $post ) {
+	static $post_classes = [];
```

Comment:
:no_entry_sign: Short array syntax is not allowed

<sub>hash: a23379381defe1d3ce93f43a95b4c3cc</sub>